pkgs/tools/admin/awscli2/default.nix

```diff
@@ -29,13 +29,13 @@ let
 in
 with py.pkgs; buildPythonApplication rec {
   pname = "awscli2";
-  version = "2.5.6"; # N.B: if you change this, check if overrides are still up-to-date
+  version = "2.7.3"; # N.B: if you change this, check if overrides are still up-to-date
```

Comment:
Did you check the overrides?

Reply:
Yes, and per https://github.com/aws/aws-cli/blob/v2/setup.cfg#L30 they are still required.